controllers/numaresourcesoperator_controller_test.go

```diff
@@ -186,7 +186,71 @@ var _ = Describe("Test NUMAResourcesOperator Reconcile", func() {
 		})
 	})
 
-	Context("with node group with MCP selector that matches more than one MCP", func() {
+	Context("with two node groups while both point to same pool using same pool specifier", func() {
```

Comment:
classifying existing tests and adding new tests should be preferably done in 2 separate commits

Reply:
addressed in last commit, thanks!

Reply:
disunited the content in the first 2 commits, thanks for catching